emuu-client/src/components/UserProfile/Profile.js

```diff
@@ -1,9 +1,43 @@
-import React from "react";
 import "./Profile.scss";
 import Feeds from "./Feeds";
 import UserInfo from "./UserInfo";
+import React, { useState } from "react";
+import "../../Firebase.js";
+import { storage } from "../../Firebase.js";
+import { ref, uploadBytesResumable, getDownloadURL } from "firebase/storage";
 
 function Profile() {
+  // Store uploaded file
+  const [file, setFile] = useState("");
+
+  //File upload
+  function handleChange(event) {
+    setFile(event.target.files[0]);
+  }
+  //If a user doesn't choose a file and tries to upload, error will appear
+  const handleUpload = () => {
+    if (!file) {
+      alert("Please upload an image first");
+    }
+
+    //Store into video folder in firebase storage
+    const storageRef = ref(storage, `/images/${file.name}`);
```

Comment:
Good use of making the storage directory as a const. This will make code easier to follow back to and read.